integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ItMiiSampleHelper.java

```diff
@@ -0,0 +1,376 @@
+  private static final String MII_SAMPLE_WLS_IMAGE_NAME_V1 = DOMAIN_IMAGES_REPO + "mii-" + CURRENT_DATE_TIME + "-wlsv1";
+  private static final String MII_SAMPLE_WLS_IMAGE_NAME_V2 = DOMAIN_IMAGES_REPO + "mii-" + CURRENT_DATE_TIME + "-wlsv2";
+  private static final String MII_SAMPLE_JRF_IMAGE_NAME_V1 = DOMAIN_IMAGES_REPO + "mii-" + CURRENT_DATE_TIME + "-jrfv1";
+  private static final String MII_SAMPLE_JRF_IMAGE_NAME_V2 = DOMAIN_IMAGES_REPO + "mii-" + CURRENT_DATE_TIME + "-jrfv2";
+  private static final String SUCCESS_SEARCH_STRING = "Finished without errors";
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+  private static String traefikNamespace = null;
+  private static String dbNamespace = null;
+  private static Map<String, String> envMap = null;
+  private static LoggingFacade logger = null;
+  private static boolean previousTestSuccessful = true;
+  private static DomainType domainTypeName = null;
+  private static ImageType imageTypeName = null;
+
+  public enum DomainType {
+    JRF,
+    WLS
+  }
+
+  public enum ImageType {
+    MAIN,
+    AUX
+  }
+
+  /**
+   * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *        JUnit engine parameter resolution mechanism
+   */
+  public static void initAll(List<String> namespaces) {
+    logger = getLogger();
+    // get a new unique opNamespace
+    logger.info("Creating unique namespace for Operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    logger.info("Creating unique namespace for Domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    logger.info("Creating unique namespace for Traefik");
+    assertNotNull(namespaces.get(2), "Namespace list is null");
+    traefikNamespace = namespaces.get(2);
+
+    assertNotNull(domainTypeName, "DomainType name is null");
+    assertNotNull(imageTypeName, "ImageType name list is null");
+
+    // install and verify operator
+    installAndVerifyOperator(opNamespace, domainNamespace);
+
+    // env variables to override default values in sample scripts
+    envMap = new HashMap<String, String>();
+    envMap.put("DOMAIN_NAMESPACE", domainNamespace);
+    envMap.put("TRAEFIK_NAMESPACE", traefikNamespace);
+    envMap.put("TRAEFIK_HTTP_NODEPORT", "0"); // 0-->dynamically choose the np
+    envMap.put("TRAEFIK_HTTPS_NODEPORT", "0"); // 0-->dynamically choose the np
+    envMap.put("WORKDIR", MII_SAMPLES_WORK_DIR);
+    envMap.put("BASE_IMAGE_NAME", WEBLOGIC_IMAGE_NAME);
+    envMap.put("BASE_IMAGE_TAG", WEBLOGIC_IMAGE_TAG);
+    envMap.put("IMAGE_PULL_SECRET_NAME", OCIR_SECRET_NAME); //ocir secret
+    envMap.put("K8S_NODEPORT_HOST", K8S_NODEPORT_HOST);
+    envMap.put("OKD", "" +  OKD);
+    envMap.put("DO_AI", String.valueOf(auxiliaryImageEnabled()));
```

Comment:
minor: the helper method is only used in one place now, so this could be 'String.valueOf(imageType == ImageType.AUX)'.

Reply:
changed